README.md

```diff
@@ -8,9 +8,15 @@
     ```
 - Set Youtube API key and Youtube playlist ID as environment variables
     ```
-    export YOUTUBE_API_KEY==your_youtube_api_key
+    export YOUTUBE_API_KEY=your_youtube_api_key
```

Comment:
The youtube API key and Google API key is the same. 

So remove this environment variable and keep only the `GOOGLE_API_KEY` 
Change @anjula-sack 's implementation according to this.

Reply:
Also please mention that the calendar has to be public

Reply:
Resolved